plugins/explore/src/components/ToolCard/ToolCard.tsx

```diff
@@ -97,7 +98,12 @@ export const ToolCard = ({ card, objectFit }: Props) => {
         )}
       </CardContent>
       <CardActions>
-        <Button color="primary" href={url} disabled={!url}>
+        <Button
```

Comment:
I was also looking at this. This should Button should actually be changed to a `@backstage/core-components` Button which has a `to` prop which is smart enough to use the react router versus href.

Reply:
Internally, the legacy button was opinionated and opened all nonlocal links in a new window. It checked the target and if it wasn't localhost-like or on the frontend base url, it was handled as external.

I think there's a bit of a two camp situation here. On the one hand, those that want the assistance with things opening automatically elsewhere. And then there are those who are basically angered by forced blank targets and want to decide for themselves whether to open locally or in a new window.

I can understand the latter camp. It can be a pain in the behind when a new window I didn't ask for opened, when a manual one is just a control-click or middle-click away if I want that.

This is especially egregious when the opening is done by Javascript instead of by a link!

I don't have a great answer here. Apparently people have been thinking the internal behavior was acceptable for the last several years, so there's that.

Maybe the annoyed power users patch it away with browser plugins and continue with their day ;)

Reply:
That is true. You can always right click on the button if you want it to open in a new tab. For something like the explore page, I personally would want a new tab for anything external. But definitely a personal preference so hard to say if this should be a baked in behaviour or something that is customized per Backstage instance.

Reply:
As long as we use the core Button and Link, we can always add a config parameter as like `app.behavior.externalLinkTarget: _blank` or something :) and then add a per-user in-app setting to override THAT some day! :)

Reply:
And arguably if we change their default behaviors some day, we should probably add that config param at the same time to let adopters choose to stay with the old behavior.

Reply:
Don't forget to use `rel="noopener"` with `target="_blank"` ([Links to cross-origin destinations are unsafe](https://web.dev/external-anchors-use-rel-noopener/))

Reply:
Updated as per suggestions.